web-registry/src/lib/ga.ts

```diff
@@ -22,7 +22,7 @@ function determineClientId(): number {
 }
 
 function initializeGA(): void {
-  var gaOptions = {};
```

Comment:
Haven't seen a `var` in a while

Reply:
> Haven't seen a var in a while

Right? it looked weird to me

>  would prefer to have the tech-debt PR separate if it is this big

This is my fault - I intended to do that, but made the mistake of branching off my original work when i was playing with this stuff, then it was hard to reconcile the git history. Again, if it's a concern i'd be happy to pull just the feature work out into another PR, then merge this in after. I'll definitely try to be more mindful moving forward either way